source/AsyncRgbLedAnalyzerSettings.h

```diff
@@ -10,18 +10,33 @@ class AsyncRgbLedAnalyzerSettings : public AnalyzerSettings
 	AsyncRgbLedAnalyzerSettings();
 	virtual ~AsyncRgbLedAnalyzerSettings();
```

Comment:
I'd like to start applying the rule of 5 to all of the user defined classes. We can start by deleting the copy & move special functions.
Quick note, you're probably aware, but if we default the destructor, we should put the "= default" in the cpp file, to ensure the destructor is compiled into the dll. Any defaulted function should be defaulted in the cpp file to ensure that it's built into the dll, and not inferred when used by the calling binary.

Reply:
I'm assuming the destructor will be on-inline (and virtual) to pin the class to the .cpp file. I haven't been deleting the copy/move operators for heap-allocated classes but I can do that here, no worries.